src/gophernicus.c

```diff
@@ -798,9 +799,11 @@ int main(int argc, char *argv[])
 	/* Remove encodings from selector */
 	strndecode(st.req_selector, st.req_selector, sizeof(st.req_selector));
 
-	/* Deny requests for Slashdot and /../ hackers */
-	if (strstr(st.req_selector, "/."))
-		die(&st, ERR_ACCESS, "Refusing to serve out dotfiles");
+	/* Deny requests for /../ hackers, deny dotfiles by default */
+	if (!st.opt_dotfile && strstr(st.req_selector, "/.")) {
```

Comment:
This bracing style is inconsistent with the rest of the code, please just remove the braces for single line ifs (applies elesewhere too).

(Apart from `options.c`)